test/conformance/tests/sriov_operator.go

```diff
@@ -1604,17 +1604,14 @@ func findUnusedSriovDevices(testNode string, sriovDevices []*sriovv1.InterfaceEx
 		stdout, _, err = pod.ExecCommand(clients, createdPod, "ip", "link", "show", device.Name)
 		Expect(err).ToNot(HaveOccurred())
 		Expect(len(stdout)).Should(Not(Equal(0)), "Unable to query link state")
-		if strings.Contains(stdout, "state DOWN") {
-			continue // The interface is not active
-		}
 		if strings.Contains(stdout, "master ovs-system") {
 			continue // The interface is not active
 		}
 
 		filteredDevices = append(filteredDevices, device)
 	}
 	if len(filteredDevices) == 0 {
-		return nil, fmt.Errorf("Unused sriov devices not found")
+		Skip("Unused sriov devices not found")
```

Comment:
Can you handle the error and skip from outside?
We don't know how this is gonna be used, and I'd prefer to keep it independent from the test behaviour.